music21/metadata/__init__.py

```diff
@@ -96,161 +184,499 @@ class Metadata(base.Music21Object):
+    'accrualPeriodicity', 'accrualPolicy', 'actNumber', 'actor', 'adapter',
+    'afterwordAuthor', 'alternativeTitle', 'animator', 'annotator', 'architect',
+    'arranger', 'artist', 'associatedWork', 'attributedComposer', 'audience',
+    'author', 'bibliographicCitation', 'calligrapher', 'cartographer',
+    'choreographer', 'cinematographer', 'collaborator', 'collectionDesignation',
+    'collotyper', 'commentator', 'commission', 'commissionedBy', 'compiler',
+    'composer', 'composerAlias', 'composerCorporate', 'conceptor', 'conductor',
+    'conformsTo', 'consultant', 'contractor', 'copyright', 'correspondent',
+    'costumeDesigner', 'countryOfComposition', 'coverage', 'creator', 'curator',
+    'dancer', 'date', 'dateAccepted', 'dateAvailable', 'dateCopyrighted',
+    'dateCreated', 'dateIssued', 'dateModified', 'dateSubmitted', 'dateValid',
+    'dedicatedTo', 'dedication', 'delineator', 'description', 'designer',
+    'dialogAuthor', 'director', 'dissertant', 'distributor', 'draftsman',
+    'editor', 'educationLevel', 'engineer', 'engraver', 'etcher', 'extent',
+    'facsimilist', 'fileFormat', 'fileNumber', 'filePath', 'filmEditor',
+    'forger', 'format', 'genericContributor', 'groupTitle', 'hasFormat',
+    'hasPart', 'hasVersion', 'host', 'identifier', 'illuminator', 'illustrator',
+    'instructionalMethod', 'instrumentalist', 'interviewee', 'interviewer',
+    'introductionAuthor', 'inventor', 'isFormatOf', 'isPartOf', 'isReferencedBy',
+    'isReplacedBy', 'isRequiredBy', 'isVersionOf', 'landscapeArchitect', 'language',
+    'librettist', 'license', 'lightingDesigner', 'lithographer', 'localeOfComposition',
+    'lyricist', 'manufacturer', 'mediator', 'medium', 'meetingOrganizer',
+    'metalEngraver', 'moderator', 'movementName', 'movementNumber', 'musician',
+    'narrator', 'number', 'opusNumber', 'orchestrator', 'originator', 'otherContributor',
+    'otherDate', 'parentTitle', 'performer', 'photographer', 'platemaker',
+    'popularTitle', 'printmaker', 'producer', 'productionPersonnel', 'programmer',
+    'projectConsultant', 'provenance', 'publisher', 'puppeteer', 'quotationsAuthor',
+    'recordingEngineer', 'references', 'relation', 'renderer', 'replaces',
+    'reporter', 'requires', 'researchTeamHead', 'researchTeamMember', 'researcher',
+    'responsibleParty', 'restager', 'reviewer', 'rightsHolder', 'scenarist',
+    'sceneNumber', 'scientificAdvisor', 'screenplayAuthor', 'scribe', 'sculptor',
+    'secretary', 'setDesigner', 'singer', 'source', 'spatialCoverage', 'speaker',
+    'standardsBody', 'storyteller', 'subject', 'surveyor',
+    'suspectedComposer', 'tableOfContents', 'teacher', 'temporalCoverage',
+    'textLanguage', 'textOriginalLanguage', 'title', 'transcriber', 'translator',
+    'type', 'videographer', 'vocalist', 'volume', 'volumeNumber', 'woodCutter',
+    'woodEngraver', 'writtenCommentator')
+
+    >>> md.allUniqueNames
```

Comment:
same here.  Maybe also showing something that is in allNames and not in allUniqueNames

Reply:
Done.